desktop/libs/notebook/src/notebook/connectors/trino.py

```diff
@@ -79,15 +79,11 @@ def create_session(self, lang=None, properties=None):
     pass
 
 
-  def get_trino_client_session(self, url):
-    catalog = urlparse(url).path.split('/')[-1]
-    user = self.user.username
-
-    return ClientSession(user, catalog)
-
-
   @query_error_handler
   def execute(self, notebook, snippet):
+    database = snippet['database']
+    query_client = TrinoQuery(self.db, 'USE ' + database)
+    query_client.execute()
```

Comment:
Why do we need to execute this for every query?

Reply:
We must execute the 'USE DB' query before every query to set the database, similar to how it's done in Hive [Hiveserver2](https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/connectors/hiveserver2.py#L331). Currently, multiple statement execution is not implemented. Therefore, we need to execute the 'USE DB' query before each query. In the future, we can consider implementing it only before the first statement.